sdl2promela/translator.py

```diff
@@ -502,11 +550,13 @@ def __generate_procedure_inline(
     builder.withName(__get_procedure_inline_name(context, procedure.name))
     blockBuilder = BlockBuilder(promelamodel.BlockType.BLOCK)
     for localVariable, localVariableTypeObject in procedure.variables.items():
-        localVariableType = resolve_asn1_type(
-            context.sdl_model.types, localVariableTypeObject[0]
-        )
+        # localVariableType = resolve_asn1_type(
```

Comment:
Why the change and why this is commented out?

Reply:
For the types like `MyBetterInteger ::= MyInteger (0..10)` the generated code was wrong:
```
MyInteger_assign_value(src, dst);
```
After these changes, the correct inline is used to assign values:
```
MyBetterInteger_assign_value(src, dst);
```